src/amber/cli/templates/error/controller.cr

```diff
@@ -0,0 +1,21 @@
+module Amber::CLI::ErrorTemplate
+  class Controller < Teeplate::FileTree
+    include Amber::CLI::Helpers
+    directory "#{__DIR__}/controller"
+
+    @name : String
+    @actions : Hash(String, String)
+    @language : String
+
+    def initialize(@name, actions)
+      @language = fetch_language
+      @actions = actions.map{|action| [action, "get"] }.to_h
```

Comment:
I don't allow any verbs other than "get". Does that seem reasonable?

Reply:
yup, seems reasonable.